rlclientlib/generic_event.h

```diff
@@ -11,7 +11,11 @@ namespace reinforcement_learning {
     using payload_type_t = messages::flatbuff::v2::PayloadType;
 
   public:
+    using action_id_t = uint64_t;
```

Comment:
[nit] The word "action" make generic event less generic (i.e. outcome_event has no actions). At the same time we actually do not really care if they are actions - just some re-usable sub-objects (it also easily can be extended to slots and maybe to some parts of shared context). Given that do we want to use more generic word here instead of action (like "object" or something). I understand that we maybe far from implementing this now, so it is purely naming question, to have some room to be able to move this direction in future.

Reply:
Good point, I'll pick a more neutral name.

Reply:
it's object_id now.